src/main/java/org/fundacionjala/at13/spaceinvaders/Bullet.java

```diff
@@ -75,8 +66,7 @@ public void isShootingToAlien() {
     }
 
     /**
-     * Description: The function displays a bullet shoot.
-     * from the Alien.
+     * Description: The function displays a bullet shoot. from the Alien.
```

Comment:
it would be better if you remove the point between short and example Description: The function displays a bullet shoot from the Alien.

Reply:
okay, already solved 😁